### ISSUE TITLE
A11Y: ToggleSmall: Provide descriptive text and proper order in toggle label elements 

### ISSUE THREAD

**Issue by rkocheis** [IBM/carbon-components-react#1183]
## Detailed description

> Describe in detail the issue you're having.

Label 'for' for ToggleSmall needs to have a descriptive text. The label should also be before the input component

===== ENVIRONMENT =====
IBM CI 162 Checkpoint Violation: IBM 4.1.2 Name, Role, Value
IBM Guidance: https://www.ibm.com/able/guidelines/ci162/name_role_value.html
Test Tool Type: DAP

===== DEFECT DESCRIPTION =====
The label element is used to explicitly associate a form control with a label. A label is attached to a specific form control through the use of the "for" attribute. Screen readers use the text in the label as a description of the attached form control.

===== EXPECTED RESULT =====
For more detail:
https://aat.w3ibm.mybluemix.net/token/334ed6c7-0a0d-480b-a007-47d1242d0ef8/1c0df9a2-5788-4caa-8052-67d0bcadfdac/archives/2018FebDeploy//doc/w3/help/en-US/idhi_accessibility_check_g1055.html

> Is this a feature request (new component, new icon), a bug, or a general issue?

bug

> Is this issue related to a specific component?

ToggleSmall, and to some extend Toggle

> What did you expect to happen? What happened instead? What would you like to see changed?

The label should be rendered before the input component, and it should have a descriptive text.

```
"carbon-components-react": {
"version": "6.18.4",
...
}
```
> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

IBM Cloud Internet Services
